NetStandard.props

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <DefineConstants>$(DefineConstants);NET_4_0;NET_CORE;</DefineConstants>
-    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
```

Comment:
❓ This condition was dropped from the new build. Will that be a problem?

Reply:
Outside of Windows Roslyn can not fully sign assemblies. I don't know if the new msbuild tooling account for that.

Reply:
Since the Travis build passed, I'm guessing it does account for it? But just a guess.